README.md

```diff
@@ -25,6 +25,7 @@ Mini Sora 开源社区定位为由社区同学自发组织的开源社区（**
 
 ## 最近更新
 
+- [Generate Impressive Videos with Text Instructions: A Review of OpenAI Sora, Stable Diffusion, Lumiere and Comparable](https://www.techrxiv.org/users/684880/articles/718900-generate-impressive-videos-with-text-instructions-a-review-of-openai-sora-stable-diffusion-lumiere-and-comparable?commit=beb4b1d1e97ec6916a48a5a32b5fa5584ba096a8)
```

Comment:
The request parameter `commit` is redundant and can be deleted. The link below does the same thing.

Reply:
> The request parameter `commit` is redundant and can be deleted. You can do the same thing to the links below.

Thank you for the reminder, I didn't even notice that the link added a commit parameter by itself.